core/src/main/java/com/homeaway/streamingplatform/StreamRegistryApplication.java

```diff
@@ -184,8 +184,6 @@ public void run(final StreamRegistryConfiguration configuration, final Environme
         environment.getApplicationContext().addServlet(PingServlet.class, "/ping");
         StreamRegistryHealthCheck streamRegistryHealthCheck = new StreamRegistryHealthCheck(managedKStreams, streamResource, metricRegistry);
 
-        // TODO: Expose this, or if it's already exposed document it correctly. (issue #46)
-        // The health check was previously documented as `http://localhost:8081/healthcheck` but that gives a 404
```

Comment:
no need to reference... its in the config-dev.yaml file.

https://github.com/homeaway/stream-registry/blob/11de62c3d4c70d8f75f6ec6452bdfc7b5d1ee588/config-dev.yaml#L4-L7